diff -uNr SugarSuite-Full-4.0.1/include/phpmailer/class.phpmailer.php SugarSuite-Full-4.0.1/include/phpmailer/class.phpmailer.php
--- SugarSuite-Full-4.0.1/include/phpmailer/class.phpmailer.php	2006-03-09 18:06:18.000000000 +0100
+++ SugarSuite-Full-4.0.1/include/phpmailer/class.phpmailer.php	2006-03-09 18:16:04.000000000 +0100
@@ -33,7 +33,7 @@
      * Sets the CharSet of the message.
      * @var string
      */
-    var $CharSet           = "iso-8859-1";
+    var $CharSet           = "UTF-8";
 
     /**
      * Sets the Content-type of the message.
