--- SugarSuite-Full-4.0.1/install/dbConfig.php~	2006-02-06 14:08:09.000000000 +0200
+++ SugarSuite-Full-4.0.1/install/dbConfig.php	2006-02-06 14:08:53.000000000 +0200
@@ -48,7 +48,7 @@
     $_SESSION['setup_db_drop_tables']       = false;
     $_SESSION['setup_db_pop_demo_data']     = false;
     $_SESSION['setup_db_username_is_privileged']    = true;
-    $_SESSION['setup_db_admin_user_name']   = 'root';
+    $_SESSION['setup_db_admin_user_name']   = 'mysql';
     $_SESSION['setup_db_admin_password']    = '';
     $_SESSION['setup_db_create_sugarsales_user']    = false;
     $_SESSION['setup_db_sugarsales_user']       = empty($sugar_config['dbconfig']['db_user_name'])  ? 'sugarcrm' : $sugar_config['dbconfig']['db_user_name'];
